test/fixtures/specs/retries.spec.js

```diff
@@ -0,0 +1,29 @@
+console.log('Retry file loaded.')
+
+describe('retry', function () {
+    this.retries(2)
```

Comment:
Why are you including Mocha timeouts here?

Reply:
I couldn't find any tests that tested the existing retry mechanisms, so I added some for those as well. I can remove them if you don't think they're necessary. I thought it couldn't hurt to have tests for retries on all three levels (test, suite, file).

Reply:
Got it but this should not be tested together with the spec retry. Having separate tests for it would make things easier I guess

Reply:
Alright. Should I move them directly into `test/spec/wdio/retries.js` (since testing test-level and suite-level retries are quite trivial), or do you want them as a separate spec in the fixtures? Or something else?